libexec/rtld-elf/malloc.c

```diff
@@ -290,6 +290,7 @@ morecore(int bucket)
 		op->ov_next = (union overhead *)(void *)((caddr_t)op + sz);
 		op = (union overhead *)(void *)((caddr_t)op + sz);
   	}
+  	op->ov_next = NULL;
```

Comment:
the commit message for this change is completely useless.  Commit messages should explain why the change was made.